lib/widgets/topic_list.dart

```diff
@@ -125,59 +126,55 @@ class _TopicList extends StatefulWidget {
 
 class _TopicListState extends State<_TopicList> with PerAccountStoreAwareStateMixin {
   Unreads? unreadsModel;
-  // TODO(#1499): store the results on [ChannelStore], and keep them
-  //   up-to-date by handling events
-  List<GetStreamTopicsEntry>? lastFetchedTopics;
+  Topics? topicsModel;
 
   @override
   void onNewStore() {
+    final newStore = PerAccountStoreWidget.of(context);
     unreadsModel?.removeListener(_modelChanged);
-    final store = PerAccountStoreWidget.of(context);
-    unreadsModel = store.unreads..addListener(_modelChanged);
+    unreadsModel = newStore.unreads..addListener(_modelChanged);
+    topicsModel?.removeListener(_modelChanged);
```

Comment:
nit: put topicsModel before unreadsModel (here and in declarations); seems like it's more fundamental to what this topic-list widget is doing